service/component-samples/reseller/src/main/java/org/fido/iot/sample/ResellerConfigLoader.java

```diff
@@ -51,11 +51,15 @@ public static String loadConfig(String property) {
       defaultConfiguration.addProperty(ResellerAppConstants.DB_USER, "sa");
       defaultConfiguration.addProperty(ResellerAppConstants.DB_PWD, "");
       defaultConfiguration.addProperty(ResellerAppConstants.DB_PORT, "8071");
+      defaultConfiguration.addProperty(
+          ResellerAppConstants.DB_TCP_SERVER,
+          "-tcp -ifNotExists -tcpPort " + ResellerAppConstants.DB_PORT);
```

Comment:
Should be defaultConfiguration.getProperty(ResellerAppConstants.DB_PORT) vs ResellerAppConstants.DB_PORT

Reply:
Fixed.